base/gfspvgmgr/virtual_group_manager.go

```diff
@@ -339,6 +336,15 @@ func (vgm *virtualGroupManager) refreshGVGMeta(byChain bool) {
 		otherSPs: otherSPList,
 	}
 	// log.Infow("list sp info", "primary_sp", primarySP, "secondary_sps", secondarySPList, "sp_map", spMap)
+	// add other SP list into health checker, except self sp, self sp should always be healthy
+	if vgm.healthChecker != nil {
+		vgm.healthChecker.cleanSPs()
```

Comment:
you can just do addSP in the same branch

Reply:
fixed